lib/src/legacy_thrift_op_store.rs

```diff
@@ -48,8 +48,10 @@ pub struct ThriftOpStore {
 }
 
 impl ThriftOpStore {
-    pub fn load(store_path: PathBuf) -> Self {
-        ThriftOpStore { path: store_path }
+    pub fn load(op_store_path: &Path) -> Self {
+        ThriftOpStore {
+            path: op_store_path.to_path_buf(),
+        }
```

Comment:
In case you were wondering, `store_path` was not a typo. I just didn't bother to say which store it was about here. I can see why you thought `store_path` was confusing, though. It's quite a big part of the commit and it's mostly unrelated, so could you move it to a separate commit?

Reply:
Oh, related to one of my other comments about naming, if we rename this to `op_store_path`, I think we should rename `store_path` in the commit backends to `commit_store_path`. I don't object to that, and I won't insist that we do it in the same PR, although I think it would most sense to do it in the same PR (or not at all).

Reply:
I've left everything as 'store_path' for now.